modules/dnn/src/layers/is_inf_layer.cpp

```diff
@@ -0,0 +1,86 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Copyright (C) 2025, BigVision LLC, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include "../precomp.hpp"
```

Comment:
please, put the link to description of the operation at onnx.ai and also specify, which opsets are supported.